scripts/configure-hypervisor.sh

```diff
@@ -133,8 +133,9 @@ check_tags()
 #    options being displayed on a single line.
 show_array()
 {
-	local -n _array="$1"
-	local action="$2"
+	local action="$1"
+	local _array=("$@")
+	_array=("${_array[@]:1}")
```

Comment:
Wow, black magic! :smile: Is this to overcome the centos bash version issue? Ah yes, I see from 88119f1fa3a4aa8ca8a262e1e759cbcf4e30bd7a that it was. Could you therefore add a "Fixes #27" comment to that commit for clarity?